### PR TITLE
Feature/fixes

### PR DESCRIPTION
**Pull request by DimarrWS**

### CODE REVIEW

File: mend_import_sbom/import_sbom.py [hunk 1]
```diff
@@ -575,16 +583,22 @@ def upload_to_mend(upload):
...
+        '''
         data = f"type=UPDATE&updateType={args.update_type}&agent={AGENT_INFO['agent']}&" \
                f"agentVersion={AGENT_INFO['agentVersion']}&token={args.ws_token}&userKey={args.ws_user_key}&" \
                f"product={args.ws_product}&timeStamp={ts}&diff={json_prj}"
+               
+               The different Agent names and versions are not acceptable for current API 1.4 method
+        '''
+        data = f"type=UPDATE&updateType={args.update_type}&agent=fs-agent&" \
+               f"agentVersion=''&token={args.ws_token}&userKey={args.ws_user_key}&" \
+               f"product={args.ws_product}&timeStamp={ts}&diff={json_prj}"
```
Comment (JRMendDemo) [inline mend_import_sbom/import_sbom.py:601]: This was originally accounted for in the code by not including ps- at the tool level. I think we still want this to show import-sbom.

Comment (DimarrWS) [reply mend_import_sbom/import_sbom.py:601]: Unfortunately, when you create a request with an agent name  different to "fs-agent" - update-request is impossible to upload. Our UI gives an error. But in other places I use AGENT_INFO, just here removed it.

Comment (JRMendDemo) [reply mend_import_sbom/import_sbom.py:601]: It should work with "import-sbom" just not "ps-import-sbom" as it was working previously.

File: mend_import_sbom/import_sbom.py [hunk 2]
```diff
@@ -658,8 +672,16 @@ def main():
...
             hdr = f'\n{len(hdr_title) * "="}\n{hdr_title}\n{len(hdr_title) * "="}'
             logger.info(hdr)
-
             log_obj_props(args, "Configuration:")
+            data = json.dumps(
+                {
+                    "requestType": "getOrganizationDetails",
+                    "orgToken": args.ws_token ,
+                    "userKey": args.ws_user_key
+            }
```
Comment (JRMendDemo) [inline mend_import_sbom/import_sbom.py:681]: Agent info needs to be added to this call.

Comment (DimarrWS) [reply mend_import_sbom/import_sbom.py:681]: Added into call_api function

**Review by JRMendDemo — APPROVED** [review]

**Pull request merged**